src/app/tree-grid/tree-grid-tree-filter-view/tree-grid-tree-filter-view.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="grid__wrapper">
+    <igx-tree-grid igxPreventDocumentScroll  #treegrid1 [data]="data" [autoGenerate]="false" height="600px" width="100%"
+        [allowFiltering]="true" filterMode="excelStyleFilter" [filterStrategy]="filterStrategy"
+        primaryKey="ID" foreignKey="ParentID">
+
+        <igx-column field="Name" header="Product Name" [dataType]="'string'" [sortable]="true" [disableHiding]="true">
+        </igx-column>
+        <igx-column field="UnitPrice" header="Unit Price Category" dataType="string" [sortable]="true" [disableHiding]="true">
```

Comment:
I think we could try the `currency` dataType for the UnitPrice column. I am not sure how it would work with the empty value for the parent records. If this doesn't work we could use the same cell template as in the other tree grid filtering samples.